AppCenter/AppCenterTests/MSAppCenterIngestionTests.m

```diff
@@ -65,9 +65,6 @@ - (void)setUp {
 
 - (void)tearDown {
   [super tearDown];
-  [self.deviceMock stopMocking];
-  [self.httpClientMock stopMocking];
-  [self.reachabilityMock stopMocking];
```

Comment:
As discussed, comment with TODO instead of deleting

Reply:
As per [this comment](https://github.com/microsoft/appcenter-sdk-apple/pull/1996#discussion_r389114830), reverting back.